### PR TITLE
[APPS-2160] migration of dependency from moment to date-fns in custom-tasks-filters.e2e.ts

### PR DESCRIPTION
**Pull request by pkunduGL**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://alfresco.atlassian.net/browse/APPS-2160

### CODE REVIEW

File: e2e/process-services/tasks/custom-tasks-filters.e2e.ts [hunk 1]
```diff
@@ -64,8 +64,8 @@ describe('Start Task - Custom App', () => {
     const totalNrOfPages = 'of 4';
     const currentDateStandardFormat = DateUtil.formatDate('YYYY-MM-DDTHH:mm:ss.SSSZ');
     const currentDate = DateUtil.formatDate('MM/DD/YYYY');
-    const beforeDate = moment().add(-1, 'days').format('MM/DD/YYYY');
-    const afterDate = moment().add(1, 'days').format('MM/DD/YYYY');
+    const beforeDate = format(addDays(new Date(),-1), 'MM/dd/yyyy');
```
Comment (arohilaGL) [inline e2e/process-services/tasks/custom-tasks-filters.e2e.ts:67]: Suggestion: you can use subDays()  to subtract days instead of adding -1.

Comment (pkunduGL) [reply e2e/process-services/tasks/custom-tasks-filters.e2e.ts]: Review comments resolved

File: e2e/process-services/tasks/custom-tasks-filters.e2e.ts [hunk 2]
```diff
@@ -64,8 +64,8 @@ describe('Start Task - Custom App', () => {
     const totalNrOfPages = 'of 4';
     const currentDateStandardFormat = DateUtil.formatDate('YYYY-MM-DDTHH:mm:ss.SSSZ');
     const currentDate = DateUtil.formatDate('MM/DD/YYYY');
-    const beforeDate = moment().add(-1, 'days').format('MM/DD/YYYY');
-    const afterDate = moment().add(1, 'days').format('MM/DD/YYYY');
+    const beforeDate = format(subDays(new Date(),1), 'MM/dd/yyyy');
+    const afterDate = format(addDays(new Date(),1), 'MM/dd/yyyy');
```
Comment (rbahirsheth) [inline e2e/process-services/tasks/custom-tasks-filters.e2e.ts:68]:
```suggestion
    const afterDate = format(addDays(new Date(), 1), 'MM/dd/yyyy');
```

**Review by rbahirsheth — APPROVED** [review]

**Review by arohilaGL — APPROVED** [review]

**Comment by eromano** [comment]
@pkunduGL @arohilaGL @rbahirsheth can you follow the right convention for the PR title and add information in the descriptions as all the other PRs of the repo? thanks

**Comment by eromano** [comment]
@pkunduGL @arohilaGL @rbahirsheth can you follow the right convention for the PR title and add information in the descriptions as all the other PRs of the repo? thanks

**Review by DenysVuika — APPROVED** [review]
please update the PR title/description according to the standards

**Pull request merged**